shared/cfg/guest-hw.cfg

```diff
@@ -141,6 +141,11 @@ variants:
         no ioquit
         image_format = qed
         check_image = yes
+    - nvme:
```

Comment:
I haven't seen a necessary reason to introduce such a new format, would you please clarify on that? thanks!

Reply:
Talked with @CongLi,  we could not consider it in this PR temporarily. Let's transfer it to the next PR after the information is confirmed.